### PR TITLE
Format for docs integration

### PR DESCRIPTION
**Pull request by meaghanfitzgerald**
## Why this should be merged

We should add this page to the AWM section of the docs. Some of the links were pointing to older versions of the coreth repo so i fixed that too.  

## How this works

edited `precompile/contracts/warp/README.md`
[Docs PR](https://github.com/ava-labs/avalanche-docs/pull/1646)

## How this was tested

View here: https://meag-awm-evm-integration.avalanche-docs.pages.dev/build/cross-chain/awm/evm-integration

### CODE REVIEW

**Review by darioush — COMMENTED** [review]
I think we should drop the line numbers if we are linking to master

File: precompile/contracts/warp/README.md [hunk 1]
```diff
@@ -73,11 +71,11 @@ The `blockchainID` in Avalanche refers to the txID that created the blockchain o
 
 ### Predicate Encoding
 
-Avalanche Warp Messages are encoded as a signed Avalanche [Warp Message](https://github.com/ava-labs/avalanchego/blob/v1.10.4/vms/platformvm/warp/message.go#L7) where the [UnsignedMessage](https://github.com/ava-labs/avalanchego/blob/v1.10.4/vms/platformvm/warp/unsigned_message.go#L14)'s payload includes an [AddressedPayload](../../../warp/payload/payload.go).
+Avalanche Warp Messages are encoded as a signed Avalanche [Warp Message](https://github.com/ava-labs/avalanchego/blob/master/vms/platformvm/warp/message.go#L7) where the [UnsignedMessage](https://github.com/ava-labs/avalanchego/blob/master/vms/platformvm/warp/unsigned_message.go#L14)'s payload includes an [AddressedPayload](https://github.com/ava-labs/avalanchego/blob/master/vms/platformvm/warp/payload/payload.go).
```
Comment (meaghanfitzgerald) [inline precompile/contracts/warp/README.md:75]:
```suggestion
Avalanche Warp Messages are encoded as a signed Avalanche [Warp Message](https://github.com/ava-labs/avalanchego/blob/master/vms/platformvm/warp/message.go) where the [UnsignedMessage](https://github.com/ava-labs/avalanchego/blob/master/vms/platformvm/warp/unsigned_message.go)'s payload includes an [AddressedPayload](https://github.com/ava-labs/avalanchego/blob/master/vms/platformvm/warp/payload/payload.go).

```

**Comment by meaghanfitzgerald** [comment]
> I think we should drop the line numbers if we are linking to master

done

**Review by darioush — APPROVED** [review]

File: precompile/contracts/warp/README.md [hunk 2]
```diff
@@ -1,18 +1,16 @@
-# Avalanche Warp Messaging
+# Integrating Avalanche Warp Messaging into the EVM
```
Comment (ceyonur) [inline precompile/contracts/warp/README.md:1]: nit: something shorter like?
```suggestion
# Avalanche Warp Messaging in the EVM
```

Comment (meaghanfitzgerald) [reply precompile/contracts/warp/README.md:1]: descriptive titles help with search engine optimization, but either title works

**Review by ceyonur — APPROVED** [review]

**Pull request merged**